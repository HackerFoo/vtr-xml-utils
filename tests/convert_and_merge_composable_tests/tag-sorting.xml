<?xml version='1.0' encoding='utf-8'?>
<pb_type blif_model=".subckt blifmodel" name="test">
  <input name="b" num_pins="2"/>
  <input name="m" num_pins="1"/>
  <input name="a" num_pins="2"/>
  <output name="bout" num_pins="4"/>
  <output name="aout" num_pins="4"/>
  <delay_constant in_port="b" max="30e-12" out_port="bout"/>
  <delay_constant in_port="b" max="30e-12" out_port="aout"/>
  <delay_constant in_port="a" max="30e-12" out_port="aout"/>
  <delay_constant in_port="a" max="30e-12" out_port="bout"/>
  <delay_constant in_port="m" max="10e-12" out_port="aout"/>
</pb_type>
