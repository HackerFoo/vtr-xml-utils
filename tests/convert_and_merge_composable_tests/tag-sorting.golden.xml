<?xml version="1.0"?>
<pb_type blif_model=".subckt blifmodel" name="test">
  <input name="a" num_pins="2"/>
  <input name="b" num_pins="2"/>
  <input name="m" num_pins="1"/>
  <output name="aout" num_pins="4"/>
  <output name="bout" num_pins="4"/>
  <delay_constant in_port="test.a" max="30e-12" out_port="test.aout"/>
  <delay_constant in_port="test.b" max="30e-12" out_port="test.aout"/>
  <delay_constant in_port="test.m" max="10e-12" out_port="test.aout"/>
  <delay_constant in_port="test.a" max="30e-12" out_port="test.bout"/>
  <delay_constant in_port="test.b" max="30e-12" out_port="test.bout"/>
</pb_type>
